docs/resources/permissions.md

```diff
@@ -718,3 +730,26 @@ The resource permissions can be imported using the object id
 ```bash
 $ terraform import databricks_permissions.this /<object type>/<object id>
 ```
+
+### Import Example
```

Comment:
```suggestion
```

merge with previous section. 

please use https://registry.terraform.io/tools/doc-preview to preview changes

Reply:
I think it looks better as a subsection rather than just having it there. What do you think?

Subsection preview:
<img width="901" alt="Screen Shot 2022-07-14 at 4 50 43 PM" src="https://user-images.githubusercontent.com/87999496/179120099-3bd0aceb-19f2-4ea2-8f9b-bb9c570c9158.png">

Merged section preview:
<img width="919" alt="Screen Shot 2022-07-14 at 4 54 29 PM" src="https://user-images.githubusercontent.com/87999496/179120083-1e160a1c-2d1f-429f-a7b2-dcda7679481d.png">